lib/state/simperium/middleware.ts

```diff
@@ -155,10 +155,12 @@ export const initSimperium = (
       return;
     }
 
-    dispatch({
-      type: 'SET_ANALYTICS',
-      allowAnalytics: !!updatedEntity.analytics_enabled,
-    });
+    if (updatedEntity.analytics_enabled !== getState().data.analyticsAllowed) {
```

Comment:
Because Simperium may use non-`true` truthy values we need to coerce to Boolean here with `!!updatedEntity.analytics_enabled`